### ISSUE TITLE
Feature Request for ALOpsAppPublish: Allow to only publish the "Test Runner" app

### ISSUE THREAD

**Issue by DanielGoehler** [HodorNV/ALOps#215]
It would be nice, if **ALOpsAppPublish** could also only publish the "Test Runner" app. It takes up 
approx. one minute to pubish all test apps vs. a few seconds, if only the Test Runner gets published.

This could be achived by considering install_al_app_names as App Filter...

```yaml
- task: ALOpsAppPublish@1
      displayName: 'ALOps App Publish'
      inputs:
        installaltesttool: true
        install_al_app_names: Test Runner #only publishes Test Runner App
```
... or by allowing to publish apps like **C:\Applications\TestFramework\TestRunner\Microsoft_Test Runner.app** from the file system of the local docker. I have not been able to get this to work.

**Comment by waldo1001** [comment]
Thing is, ALOps helps people not forgetting the rest of the apps, in case of certain versions of BC.  This now has the drawback with what you say.  Us removing that "help", might end up in failing pipelines.. .  So we'll have to think about this on what the best approach would be ...

**Comment by DanielGoehler** [comment]
I personally would put the apps in the tooltip of **installaltesttool** itself. If **install_al_app_names** can't be changed or removed, how about a new parameter **altesttool_filter**?

**Comment by AdminHodor** [comment]
Dear @DanielGoehler ,

What we could do, without breaking other peoples pipelines is introduce a new boolean parameter "strict_app_names" which we'll default to FALSE. 
When set to TRUE, we'll skip all the 'smart' stuff, and just go forward with the input supplied by the pipeline.

Would that work for you?

Kind regards,

**Comment by DanielGoehler** [comment]
Perfect

**Comment by AdminHodor** [comment]
Will do and keep you posted on this.

**Comment by AdminHodor** [comment]
Dear @DanielGoehler ,

We just released v1.435.1856 which should tackle this issue. There's a new boolean parameter [strictappnames] you can enable to get the behavior you want.
Please note it could take a couple of pipeline runs for DevOps to pickup the new version.

Kind regards,

**Comment by DanielGoehler** [comment]
Hi @AdminHodor, thanks! Works perfect. Best regards.
